src/electron/views/renderer-initializer.ts

```diff
@@ -440,6 +440,11 @@ getPersistedData(indexedDBInstance, indexedDBDataKeysToFetch).then(
             null,
         );
 
+        const startTesting = () => {
+            windowStateActionCreator.setRoute({ routeId: 'resultsView' });
+            windowFrameActionCreator.maximize();
```

Comment:
Issue #2147 asserts that we don't always want to maximize this, and it might make sense to put the window sizing logic into the code that's associated with the resultsView. It's non-blocking, and we can certainly refactor it later.

Reply:
Ok, got it! Thanks, Dave